uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-widget/leave-widget-absence-types-available-balance.component.js

```diff
@@ -26,6 +25,8 @@ define([
     var entitlements;
     var vm = this;
 
+    vm.leaveEntitlements = [];
```

Comment:
but i was wondering, do you really need a new list at all? Why don't you simply enhance the list of absence types, without having to create a new list? You can simply have a `vm.absenceTypes` list, where each absenceType has also the entitlement data in it. I think it would make things cleaner